Assignment2/juanked-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,55 @@
+{
+    "@context": {
+        "vocab": "http://upmsemanticweb.com/students#",
+        "fullName": "vocab:fullName",
+        "hometown": "vocab:hometown",
+        "age": "vocab:age",
+        "almaMater": "vocab:almaMater",
+        "familiarsNames": "vocab:familiarsNames",
+        "father": "familiarsNames:father",
+        "mother": "familiarsNames:mother",

```

Comment:
These attribute definitions work, but semantically they aren't ok. The best way is to use a prefix and add the name of the concept/property you want to use e.g. "mother":"yourvocab:mother".
You don't need to change them, but remember to provide good definitions for the future.